scripts/installUbuntuDeps.sh

```diff
@@ -94,16 +94,29 @@ install_apt_deps(){
   sudo apt-get update -y
   check_version
   echo "Installing gcc $gcc_version"
-  sudo apt-get install -qq git make gcc-$gcc_version g++-$gcc_version python3-pip libssl-dev cmake pkg-config liblog4cxx-dev rabbitmq-server mongodb curl autoconf libtool automake -y
+  sudo apt-get install -qq git make gcc-$gcc_version g++-$gcc_version python3-pip libssl-dev cmake pkg-config liblog4cxx-dev rabbitmq-server curl autoconf libtool automake -y
   sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-$gcc_version 60 --slave /usr/bin/g++ g++ /usr/bin/g++-$gcc_version
   echo "done"
   
 
   sudo chown -R `whoami` ~/.npm ~/tmp/ || true
 }
 
+install_mongodb(){
+  if [ -d $LIB_DIR ]; then
+    echo "Installing mongodb-org from tar"
+    sudo apt-get install -y libcurl4 openssl liblzma5
+    wget -P $LIB_DIR https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2004-4.4.4.tgz
+    tar -zxvf $LIB_DIR/mongodb-linux-x86_64-ubuntu2004-4.4.4.tgz -C $LIB_DIR
+    sudo cp $LIB_DIR/mongodb-linux-x86_64-ubuntu2004-4.4.4/bin/* /usr/local/bin/
```

Comment:
would symbolic links work? I don't particularly like copying the binaries there when they may change.
Also, this may create new problems with anyone that already has a mongo version installed. We could consider using a suffix or a preffix for the binaries. For every other thing I think we either use a ubuntu package or we keep it in `LIB_DIR` without affecting `/usr/bin` or `/usr/lib`

Reply:
I would also add a way to clean everything up in the `cleanup` function

Reply:
Done!! Thanks for the suggestion :)